### ISSUE TITLE
How to restore data from backup

### ISSUE THREAD

**Issue by yattagit** [CrunchyData/postgres-operator#1881]
Hi,
I'm using PGO 4.4.0 version and trying to backup Postgre DB cluster and restore data again. 
I'm using Azure WS environment.

I successfully taking backup with "pgo backup hacluster" command.

After that I can see the backup files are like below;

```
bash-4.2$ pgo show backup hacluster
cluster: hacluster
storage type: local
stanza: db
    status: ok
    cipher: none

    db (current)
        wal archive min/max (12-1)

        full backup: 20200913-185117F
            timestamp start/stop: 2020-09-13 18:51:17 +0000 UTC / 2020-09-13 18:51:45 +0000 UTC
            wal start/stop: 000000010000000000000002 / 000000010000000000000002
            database size: 31.1MiB, backup size: 31.1MiB
            repository size: 3.7MiB, repository backup size: 3.7MiB
            backup reference list:

        incr backup: 20200913-185117F_20200913-190743I
            timestamp start/stop: 2020-09-13 19:07:43 +0000 UTC / 2020-09-13 19:07:46 +0000 UTC
            wal start/stop: 000000010000000000000009 / 000000010000000000000009
            database size: 31.1MiB, backup size: 2.8MiB
            repository size: 3.7MiB, repository backup size: 322.7KiB
            backup reference list: 20200913-185117F
```

And pods are like below;

```
PS C:\Users\user> kubectl get pods
NAME                                             READY   STATUS      RESTARTS   AGE
backrest-backup-hacluster-n52sq                  0/1     Completed   0          2m21s
hacluster-backrest-shared-repo-c5d4bb7cd-q7xmx   1/1     Running     0          12h
hacluster-mqry-847dc47b76-dmrs4                  1/1     Running     0          10h
hacluster-smfz-86bf7d9f9d-764cj                  1/1     Running     0          10h
hacluster-stanza-create-7gpzl                    0/1     Completed   0          12h
pgo-client                                       1/1     Running     1          12h
pgo-deploy-d5dcx                                 0/1     Completed   0          14h
postgres-operator-5d5ff486c7-bdcqt               4/4     Running     1          14h
```

So I found that if I want to restore data from a backup, I can use the command below;

`pgo create cluster restoredcluster --backup-path=/somebackup/path --backup-pvc=somebackuppvc --secret-from=mycluster`

But I don't know how can I get "--backup-path" variable?

Also if I use "--backup-pvc", it throws "unknown flag: --backup-pvc" error.

What should I write for this variable? How can I get this path data?

Thanks!

**Comment by jkatz** [comment]
Please review the documentation, this topic is covered fairly extensively:

- [https://access.crunchydata.com/documentation/postgres-operator/latest/architecture/disaster-recovery/](https://access.crunchydata.com/documentation/postgres-operator/latest/architecture/disaster-recovery/)
- [https://access.crunchydata.com/documentation/postgres-operator/latest/pgo-client/common-tasks/#disaster-recovery-backups-restores](https://access.crunchydata.com/documentation/postgres-operator/latest/pgo-client/common-tasks/#disaster-recovery-backups-restores)

**Comment by yattagit** [comment]
Hi Jonathan,
Thanks for the links, I checked both of them but still have some questions, I would be appreciate if you help.

If i run the command below, command is creating "testpvc" pvc successfully.

`pgo backup hacluster --backup-type="pgdump" --pvc-name="testpvc"`

But I can't able to restore data from this PVC. Tried lots of variation of commands but still same. Which command should I use?

Thanks!
